### ISSUE TITLE
Feature Request: Add a Security Vulnerability Disclosure page

### ISSUE THREAD

**Issue by johnwalicki** [open-horizon/open-horizon.github.io#351]
### Is your feature request related to a problem? Please describe.

The Open Horizon documentation should include a page which describes our responsible Security Disclosure procedure and link to the .github/SECURITY.md
It may also provide details from that SECURITY.md page.
```
Please report any potential security issues to `emailaddress@example.com`. This will notify the core project team who will respond accordingly.
```
Our security disclosure procedure implementation will have a way to post to LF managed secure mailing list. The Open Horizon maintainers will be subscribed to this private list.

@joewxboy is working with the LF infrastructure team to establish that mailing list.

**Comment by johnwalicki** [comment]
This page will trail the official SECURITY.md being defined by 
https://github.com/open-horizon/.github/issues/20

**Comment by johnwalicki** [comment]
Closed by #362
